Makefile

```diff
@@ -208,6 +208,7 @@ test_lint:
 format:
 	find . -name '*.go' -type f -not -path "./vendor*" -not -path "*.git*" -not -path "./client/lcd/statik/statik.go" | xargs gofmt -w -s
 	find . -name '*.go' -type f -not -path "./vendor*" -not -path "*.git*" -not -path "./client/lcd/statik/statik.go" | xargs misspell -w
+	find . -name '*.go' -type f -not -path "./vendor*" -not -path "*.git*" -not -path "./client/lcd/statik/statik.go" | xargs goimports -w -local github.com/cosmos/cosmos-sdk
```

Comment:
Is `goimports` an automatically included binary with Go? I can't recall. If not, we'll have to make sure it's installed via the Makefile.

Reply:
yeah, we need to ensure that